custom_liferay_overwrite.css

```
@@ -1,37 +1,57 @@
 /* Makes Navigation bar go over entire page width 
 and Removes Side margins from Liferay Homepage */
+
 #wrapper {
   padding: 0;
   margin: inherit;
 }
 
 /* Moves logo and site number to the right */
-#banner > #heading {
+
+#banner>#heading {
   padding: 0px 5px 0px 5px;
 }
 
 /* logo sizing */
+
 #heading .custom-logo {
   height: 5rem;
 }
 
-#heading .logo.custom-logo > img{
+#heading .logo.custom-logo>img {
   width: inherit;
   height: inherit;
   object-fit: contain;
 }
 
+/* Changes background color and font-size for navbar */
+
+#navigation .navbar-inner {
+  background-color: var(ncct-dark) !important;
+}
+
+/* Changes background color for active navbar element*/
+
+#navigation .nav>.active>a {
```

```suggestion
.aui #navigation .nav>.active>a {
```
Use the same selector as in the liferay theme. This assures correct overwrite.